### PR TITLE
Enables upgrading and supercooling servers to produce more points.

### PR DESCRIPTION
**Pull request by Buggy123**
Servers will now produce 33% more points per tier of parts, double with triphasic scanners. They also linearly produce more points the colder they are, double at the lowest possible temperature. This can stack to a maximum of 4x point generate rate. The default, round-start point generation has not been changed.

Active servers must be cooled in order to function, at room temperature they will not generate any points at all. Additionally, servers now utilize cooperative computing; the points generated are a sum of the active research servers. You won't generate more points by making more servers, however if any actives servers are not upgraded or poorly cooled you will generate less points proportional to the state of the servers.

For example, 1 default server at the normal temperature of a freezer will generate 2,500 points a minute. So will 2, or 10 under the same conditions. 1 server in such a state, and 1 sitting at room temperature will generate 1,250 points per minute; the poorly cooled server will bog down the other server as they try to work together. One fully upgraded server and one normal server will generate 1.5x as many points as normal. And so on.

I would change it such that building more servers generates more points, but Kor explicitly forbade that and closed the last PR which did exactly that. If you want that to work, change his mind.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
this is exploitable

1. build server room
2. make hole to spess in 1 floor
3. congratulations u got da points

**Comment by Buggy123** [comment]
1: There is literally no way to increase point generation as-is except bombs, and no one does bombs because its difficult and annoying and not that great, and with fusion disabled its even harder. We were told that we were going to get new ways to generate points, and we haven't.

2: Old science took 5 minutes and then it was completely done, new science takes a hour and a half with no way to rush it on a server where 50% of rounds are 30 minutes or less.

3: Science has already been nerfed into the ground, stomped on, kicked in the groin, shot repeatedly, and then pissed on. Even if this flat out buffed point generation by 2x, no strings attached, I still think it would be acceptable. As is, scientists have to put in some sort of effort every round to get the bonus.

**Comment by deathride58** [comment]
Rnd bitcoin mining sim now?

When will we be able to use modular consoles/tablets/laptops to contribute to RND point production, with the downside being that they heat up faster and are more sensitive to heat (to the point of literally exploding) compared to normal RND servers?

Players should definitely be rewarded for successfully constructing an RND server warehouse, though. You could probably nerf the base generation rate back to 1k per minute and cap it there, but make every additional server add an extra 25 points per minute onto that base generation rate and cap. With each server adding 25 ppm, it'd take a total of 42 cooled, non-room-temp servers in order to reach a rate and cap of 2k points, which, so long as RND servers produce a significant amount of heat, would require quite a bit of cooperation between research, engineering, and atmos.

The issue with your old PR is that extra servers made the generation rate scale way too sharply, which made it way too easy to skip the timegate that techwebs is supposed to be. You should probably make sure upgrades and supercooled environments don't bump the generation rate up too harshly while you're checking back on changes to this PR. And while you're making changes to the PR, keep in mind that techwebs is balanced around a completion time of 1 hour, 30 minutes, so touching the point generation rate can and will have a drastic effect on RND's balance as a whole.

tl;dr: if you're gonna add balancing that makes it reminiscent of bitcoin mining, why not go all the way and make it possible for players to conduct large-scale mining operations? just keep in mind that techwebs is meant to be a timegate to ensure endgame stuff remains endgame stuff

**Comment by Buggy123** [comment]
I mean I don't disagree with you on any of those points, but I am not the person for that job. This PR took **two days** of on/off work.

**Comment by KorPhaeron** [comment]
Why did you spend two days making a way to nullify the RD timegate right after I said the timegate was the most functional part of techwebs

Being able to reach x4 point gen just sounds absurd

**Comment by Buggy123** [comment]
That requires max upgrades which themselves require a fair bit of research. Moreover, you said that the problem was **skipping** the timegate, at best this reduces it, and to get 4x you need to research high-tier parts first so by the time you get it it'll be 30-40 minutes in.

As per the timegate being the best part of techwebs, or even a _good_ part, I disagree entirely. There are no interactive elements of any sort to it, if you want to get something and it's gated behind research there is _NOTHING_ you can do but wait while the round slowly decays around you. On top of that, the speed of it is set so low that most rounds devolve before you get whatever you want; even if you don't max it, by the time you research the prerequisites and the obligatory advanced mining tech, the shuttle could already be called.

On top of this, the time to get a neat gadget is usually _not_ "1 minute before the shuttle leaves". Getting something cool, and then having it for 5-10 minutes is not good, you've already a large portion of the opportunities where it would be useful.

And also, if the particular thing you want is a mech? Fuck you. Fuck your family. Fuck your dog. Mechs are gated behind gated gates with titanium steel locks, rigged with boobytraps, and surrounded by mines. If your goal as a roboticist is "make a good combat mech" you should suicide roundstart and then remove TGStation from your favorites list because you'll never, ever get it.

Lets compare literally all of the jobs real quick, and see how their goodies compare to science's:

Captain: Gets most of their goodies round start, takes whatever other goodies they want from other departments. Starts with practically everything Has access to the security hud system which has many uses.

Head of Personnel: Gets most of their goodies round start, can get more through cargo 5-10 minutes in unless they want a lot. Starts with all access among other things. Has access to the security hud system which has many uses.

Head of Security: Starts with armor, a unique gun nonlethal/lethal gun, many other guns, and a personal space suit. Has access to the security hud system which has many uses.
Warden: Starts with armor, powerful nonlethal guns, can easily get lethal guns. Has access to the security hud system which has many uses.

Security Officer: Starts with multiple melee stun weapons and ranged stun weapons. Has access to the security hud system which has many uses.

Detective: Starts with a powerful 1-hit stun revolver, can potentially get more from security. Has access to the security hud system which has many uses.

Chief Medical Officer: Starts with powerful healing chems, access to numerous medkits of all kinds, virology, chemistry, genetics, and a space suit.

Chemist: Starts with everything or nearly everything required to make stun-eliminating drugs, healing chemicals, chemical space-suits, death grenades, explosives, and more, all with 10 minutes of work or less.

Geneticist: Can find most/all superpowers within 20-30 minutes depending on skill, includes things such as stun immunity and extremely high melee damage, telekinesis, cold immunity, and xray vision. Access to various medkits.

Virologist: Can make extremely deadly or annoying viruses within 5-10 minutes roundstart, capable of treating nearly any disease roundstart. Access to various medkits.

Medical Doctor: Access to various medkits.

Chief Engineer: Starts with a fireproof and radproof suit with built-in jetpack, RCDs, construction material, mesons, and more.

Station Engineer: Starts with suits, RCDs, construction materials, and more.

Atmospheric Technician: Starts with fireproof spacesuits and firesuits, various construction materials, and sometimes an RCD.

Quartermaster: Starts with access to a very wide selection of items including guns and armor, depending on various factors and work put in. 5-10 minutes to a modest number of guns is feasible.

Shaft Miner: Roundstart access to melee-resistant armor, unlimited ammo limited range guns, 20 brute weapons, heal-all pens, and emergency don't-die-in-space capsules. Within 20-30 minutes can acquire: riot/hardsuit/firesuit combos with jetpacks, instant complete heal items, better ranged unlimited ammo weapons, magic d20 dice, wands of fire, various minor spellbooks, staffs which create ultradeath tiles, staffs which turn any room into slow death, voice of god, items which make you invulnerable for a few seconds, 40 brute weapons, "make everyone on the shuttle kill each other": the item, "make everyone on the station kill someone you pick": the item, they get...

Cargo Technician: Starts with moderated access to a very wide selection of items including guns and armor, depending on various factors and work put in.

Bartender: Roundstart access to a stun/lethal gun and powerful stun chemicals.

Cook: Starts with various decently-powerful melee weapons.

Botanist: Can make 50-brute melee weapons no one else can pick up, 10-second throwable slips, throwable acid death, spacevines, extremely powerful healing chemicals, and more within 10-30 minutes depending on skill.

Curator: Can speak any language, gets a instant catgirl stun, and no one ever considers them a credible threat.

Chaplain: Gets a selection of powerful melee weapons, as well as powerful anti-cult and anti-magic tools including holy water and the bible.

Clown: Honk

Mime: Roundstart access to a invisible and unbreakable wall spell.

Janitor: Roundstart slip immunity, as well as a massive array of slips, and a legitimate excuse that allows them to slip people regularly without being suspect.

Lawyer: Can access security to various degrees, no one takes them seriously.

Assistant: Free-reign access to maintenance and all the items contained therein, doesn't have to do anything so they can do exactly what they want.

And finally:

Scientist: Access to circuits within 5 minutes, allowing for numerous gimicky and useful things, but gated behind a higher skill and knowledge barrier than anything else in the entire game. Access to bombs which they more-or-less can never use unless they're a antag, 5-10 minutes. Access to machines once researched and built, including cloners and more, 20-30 minutes and moderately dependent on other departments. Access to a wide array of goodies in xenobiology, 30-40 minutes. Access to many of the goodies from science including flightsuits, 40 minutes-_1 hour 30 minutes_ and heavily dependent on other departments.

Roboticist: Can access at least research without much fuss, can create emaggable cyborgs, can augment eachother, 5-10 minutes. Can create mobile weapon platforms, _50 minutes+ and entirely dependent on other departments_.

Research Director: Access to both of the above jobs, as well as a round-start hardsuit and unique armor.


See, now, scientists and roboticists? They get some nice stuff, they're up there as one of the better jobs in that regard. But the wait?
Longer.
Than.
Literally.
# ANY.
Other.
Job.

See, there's a inherent issue there that goes beyond just "they have to wait for longer". The game is BUILT with shorter jobs in mind, it _has_ to be because literally _every other job is short._ So we get 30 minute shuttle calls, we get 20 minutes until OPS storm the station, we get 40 minutes of gluing your eyeballs to the R&D servers only for them to steal the boards _AND_ your eyeballs. We get zombies 30 minutes in, we get megafauna roaming the station 30 minutes in, we get 5 syndiebombs blowing up the station 30 minutes in, we get plasmafloods 40 minutes in, we get random "I stubbed my toe" shuttle calls that no one bothers to recall, we get blobs 30 minutes in, revs run the station into the ground by 10 minutes, wizards summon guns and the station is in ruins in 20 minutes, cultists are rampaging with bloody bastard swords in 30 minutes...

A job intended to peak at least 10-20 minutes after the action peaks is retarded. You think scientist gets too many goodies? Fine, nerf _THAT_. But now that science's goodies are spread out throughout the station, EVERYONE has lost something, not just scientists. And gating items behind immense, noninteractive waits, instead of removing it if its bad or making it achievable if its alright, is retarded.

To summarize, techwebs the waiting game is retarded.

**Comment by XDTM** [comment]
The whole damn point of techwebs is that you have to choose what you want to develop, not get everything every round

**Comment by Buggy123** [comment]
Yes, that's where the hour and 30 minutes figure comes from, that's if you want EVERYTHING. Equipped mechs alone are more like 40 minutes, once you have the mandatory research done.

**Comment by Iamgoofball** [comment]
so what you're saying is you're mad that the main toy for scientists is something that actually requires research and development?

**Comment by kevinz000** [comment]
i'm not going to even read the essay you wrote and say "come up with ways of making points and tell them to me so i can include them in the next patch, or make them yourself." because making rnd be able to be done in 30 minutes is how you havep eople complaining it's op.

**Comment by Buggy123** [comment]
Goof:
I'm mad that the main toy for scientists is not designed properly. It's not designed with the rest of the game in mind, and it suffers greatly because of that.

kevinz:
One thing that was suggested was a interactive minigame of some sort. They suggested something cookie-clicker ish. Above all it needs to be something _interactive_ and something _meaingful_.

With something like xenobiology, everything you do has a purpose; feeding the slimes monkies, picking what slimes to breed, etc. Every action in itself accomplishes something, and those accomplishments work towards the greater goal. Pretty much every job is like this So, above all, we need to _not_ add something like, as a simplified example, a minigame where so long as you press a button at least every 10 seconds, you get 50% more point income. There's no room for skill or improvement, and pressing the button itself is meaningless, it just arbitrarily enables faster point income instead of doing something with each press.

**Comment by optimumtact** [comment]
didn't this pr get closed already

**Comment by optimumtact** [comment]
techwebs taking toys away from science is by design.

Scientists who try to play RnD the same way they did previously will find RnD boring and should find a new job.

Scientists who can find reward in things other than all of the most overpowered gismos will enjoy science.

Clearly not awesine or buggy, who still think that science printing items is their main toy when it's pretty clear we just took it away.

**Comment by optimumtact** [comment]
fuck the rest of my argument why am I bothering

**Comment by arsserpentarium** [comment]
Honestly,now every way to entertain science is valuable.
Because now after node"advanced dafa theory",science is demotivated even from idle.
I also think, that this timegate is sucks. Also sucks,wken your job is waitress at timegate. This is a reason for demotivation.

**Comment by arsserpentarium** [comment]
Also I mostly kidding about cookie clicker. I am just saying, that cookie clicker is better than what we have now.

**Comment by Iamgoofball** [comment]
>I'm mad that the main toy for scientists is not designed properly. It's not designed with the rest of the game in mind, and it suffers greatly because of that.

we've explained like 50 times why it's fine the way it is, go home

**Comment by Bawhoppen** [comment]
Don't halfass techwebs like this PR is

**Comment by KorPhaeron** [comment]
If you think the timegate is too long I wouldnt mind lowering it to an hour or whatever, but either having a timegate or not having a timegate based on whether someone printed stock parts seems silly

**Comment by KorPhaeron** [comment]
Putting the tech selection in robo/rd was not a good idea it confused people into thinking its still a science job it should have stayed bridge/rd office only

**Comment by epochayur** [comment]
remind me again why everyone is in favor of reducing an entire department to watching a number tick up? Why not make it similar to engineering in that you actually have to set something up, where you have +0 points/m at start and actually have to go make a server or something to start getting points. I don't see how actively keeping gameplay away from R&D in this Video Game designed for Fun is in anyone's best interest

**Comment by KorPhaeron** [comment]
>reducing entire department to watching numbers go up

But they still have xenobio, robotics, toxins, experimentor (sort of) and circuits 

Old RnD was a shitty timegate with no gameplay as well but the timegate failed because of items constantly being added and removed and we ended up with shit like xrays at 8 minutes and 15 minute phazons (before the anomaly requirement)

>hard mode dont discuss the thing the pr is about

**Comment by epochayur** [comment]
my point is; why make RnD nothing when it could be something? is it really just to keep people from getting good items early?

**Comment by optimumtact** [comment]
@KorPhaeron daily reminder to move it to the department head offices

**Comment by Cyberboss** [comment]
> is it really just to keep people from getting good items early?

Yes

**Comment by Buggy123** [comment]
>If you think the timegate is too long I wouldnt mind lowering it to an hour or whatever, but either having a timegate or not having a timegate based on whether someone printed stock parts seems silly

Something is better than nothing, If you're okay with that then when this closes I could make a PR which increases point generation by a flat 33%.

**Comment by XDTM** [comment]
if you make R&D timegates vary based on player skill you get wildly unbalanced equipment compared to the progression of other departments. To avoid minute 7 xrays you'd have to nerf it, and you'd end up with an RnD that powergamers could do in ~10 minutes and everyone else either takes too long or doesn't bother at all.

**Comment by Buggy123** [comment]
But every department's timegate varies based on skill. A chemist who has to look up stuff on the wiki can take 10 minutes to make any of the really fancy chems. A chemist who has it memorized can do it in 1-2 depending on how fast they press can buttons.

**Comment by XDTM** [comment]
Chemistry has no timegate besides pressing buttons. This means that every chem has to be balanced to be around at roundstart with few exceptions, which require collaboration or external factors, like rezadone, zombie powder and strange reagent.

R&D is supposed to make experimental, powerful stuff and can't rely on that model, or we'll get much less powerful items from it.

**Comment by Buggy123** [comment]
But they don't? AAdmittedly I haven't played it a lot since the nerf, but off the top of my head R&D can make:
Flightsuits
Gimmicky guns (portal guns)
Mechs (sorta)

Flightsuits and Mechs are technically a robotics thing, and gimmicky guns aren't terribly useful. Both of those also require cooperation from mining for minerals. All of the other experimental, powerful stuff was moved into other departments.

So, chemistry, roundstart, gets a shitload of healing chems, death chems up the wazoo,, explosive grenades, "melt a hole to space" foam grenades... with collaboration and external factors, they get rezadone, zombie powder, strange reagent, "fuck i can't even look at the screen without throwing up"nium, and more.

Research itself(not including xenobio, toxins, etc), roundstart, gets nothing. After 40+ minutes and with collaboration, they can get flightsuits and a portal gun, and a mech or two if robotics is willing to put in a lot of work.

They're supposed to make experimental, powerful stuff, but they don't. If all of the departmental shit was moved back to research, I'd be okay with the timegate. As is, I think we either need to buff research, or just put it out of its misery and remove it.

**Comment by KorPhaeron** [comment]
@Buggy123 

Research itself isn't a job anymore, that's the point. It's going to be a track of the entire stations progress, that everyone contributes to, that everyone gains from. 

I understand its disappointing there isn't a "job" anymore that just prints out all the coolest new toys coders have added but it was a problem for the rest of the game that adding any new item meant science would have it mass produced within 15 minutes of roundstart and the same job was constantly getting all the cool new stuff

**Comment by Buggy123** [comment]
That's part of it as well. There shouldn't be so many scientist slots anymore. 1 in xenobio is plenty, 2 is a crowd. Toxins isn't a job, its a tool for antags to blow up the station with and its completely useless outside of such. And circuits are a meme. Honestly you could just remove scientists, remove the experimentor and compact down toxins a bit to reduce the science department to half its size, give roboticists free reign, and make xenobiologist its own job.

But other than that, XDTM was making the point that they can make cool experimental stuff through research, and I was arguing that they can't. Research is not a "wait for amazing goodies" thing. It's a "wait for 30-40 minutes for literally anything, and those things are kinda meh except mechs and that's not even technically your job". There are more than a few other jobs which get access to much better stuff right off the bat.

**Comment by XDTM** [comment]
Kor's point is better, it's not a job, and shouldn't be compared to one.

**Comment by Cyberboss** [comment]
I do agree with folding in robotics

**Comment by optimumtact** [comment]
on the bright side, now that techwebs isn't funneled to a single department, we can start introducing some more powerful stuff to it.

The same concern of science getting all the endgame weapons doesn't apply as much anymore

**Comment by Bawhoppen** [comment]
Now we just need to seamlessly integrate it into other game systems, and we're golden.

**Comment by arsserpentarium** [comment]
I disagree with remowing scientists slots. Because circuits isn't a meme. And if there will be less slots, often Iwon't be able to sign up as scientist and make constructions.

**Comment by imsxz** [comment]
let's add some more powerful gear that the other departments will never print, and bored scientists stealing their lathes/explore space for the omnilathe and print them for themselves.

**Comment by ShizCalev** [comment]
>38 posts
>one vote

**Comment by Rinoka** [comment]
just going to point this out: if your idea for this is to make it so research isn't a job anymore, kor, then counterpoint: why wouldn't you want to make it a replacement to station goals in a way that actually compliments the game, too? i mean rn station goals don't get touched because they're generally irrelevant but...

my opinion on techwebs is pretty out there and available, and i've been p. direct with it, but i don't think departmental lathes are bad (even if there's the potential for abuse, it's less easy and more involved), but what i do think is that there's no positive parts of techwebs, from a game design perspective.

waiting is not and never will be a Fun Game Mechanic. timegating in a way nobody can influence without ridiculous effort is the ultimate negative Sin of game design; please review battlefront 2 for examples.

kor i get that we're trying to solve the issue of Science being potentially **Too Powerful** and i support that. and i get that techwebs are a preliminary product that's been pushed into the game in an unfinished state. but also, i think there needs to be a way - any way - for science and/or the rest of the station to reduce the timer. even if that means the initial timegate is increased and the "base rate" for full completion remains the same (1h30m), that's way more positive and player-influenced than the

**Comment by XDTM** [comment]
It should be a station effort, though, not building a couple more cheap machines.

**Comment by epochayur** [comment]
I very much agree with Rin. It doesn't have to be servers, and it definitely doesn't have to be just science pitching in, but anything is better than the literal nothing we have right now. Immutable timegates are anti-fun

**Comment by Bawhoppen** [comment]
Neither is OP R&D

**Comment by epochayur** [comment]
why does it have to be powergaming? why is having any mechanics at all to increase the points "powergaming"?

**Comment by Rinoka** [comment]
because it's the easiest counterpoint based on a well-known tendency and a belief that you can only have one or the other.

in reality, the sad thing is that techwebs rn are either tedious or broken because of what they lack.

1) it's tedious at best, unclear at worst, to figure out what the "tech chain" is. that means you don't have an understanding of what leads to what leads to what without memorization, and nobody will memorize "the best chain for x situation", they'll memorize "the most balanced/op chain for every situation". 

this is typically why games have an actual visible techweb going - without it, players are basically stranded without a wiki guide or more patience than most are willing to devote to a game where rounds last 30m and it'd take 20m to puzzle out what the fuck is going on.

this was an issue with the old system, but the new system makes it less intuitive and makes it easy to waste time if you accidentally select things that aren't useful, unlike deconning stuff, which was just a material waste more than the seconds wasted on the decon itself.

2) currently reducing the timegate just allows players to get more items in a given tree - it permits experimentation and eventual refinement into powergamey shit like before, where you've got eguns or whatever in 15 mins.

3) these two things literally work against each other. the more you permit people to experiment with the system, by reducing punishments for "useless tech" the more likely they are to find the best path to the most useful tech in the shortest period. but if you don't let people experiment, they'll probably look up a guide to understand how to do the bare fucking minimum based on someone who invested substantially more fucks, which means the system becomes cookie cutter and dull based on whatever the guide meta is.

the fact there aren't any positive reinforcers like "hey look!! you made research go faster you're a Good Player" makes it effectively poison, too, like i mentioned up there. it is a timegate with no way to de-timegate, it doesn't pretend the player has an influence on how research functions, and that makes it painful.

but really, what techwebs needs to succeed, imo?

1) more positive language. things don't actually need to be _changed_, at least not on a major scale; the final times might still end up very similar, but there needs to be more "Look At The Things You Can Do To Make This Not Suck". even if it's a holdover, i kinda recommend extending times but making it so producing unique items (ie, botany mutations, slime cores, high-cap bombs, etc) can be just passed into a wall-mounted deconstructor that lowers the timegate by boosting point gen or giving x minutes' worth of points or whatever, but only allows a unique item to be deconned once for points. even if it's just a holdover it'd go a long way for making people feel useful, and it'd help positively promote assistants/etc having a kind of weird job (unique item gatherers for science).

2) way more self-explanatory; there needs to be visual cues for what does what and leads where. the easiest place to start is to divvy everything up into meaningful categories then find summary words & color code things.

ie, all mech boards? buttons are purple, "ROBOTICS" is their tag. engineering/power stuff? yellow "POWER". i could probably brainstorm up the given categories but my POINT is that each category kind of needs to have a self-evident line of where it will lead, either by being visually presented that way _as a tech web_ or otherwise having a clear system for "PICK ME IF YOU WANT TO DO X".

personally, i think the color system might be a good & easy first step, esp. if it's possible to mix colors (ie, special mech power core could be half purple half yellow - basically just showing "what's next".) mix it w/ a tiered system of upgrades and there'd be interesting potential for a low-effort information-dense display.

i feel like there's more that could be done but these are two things i think would be low-effort and higher priority towards improving the system, rather than just raw improvements to point gen like this pr.

**Comment by Buggy123** [comment]
So... I can... change it it if its not acceptable as is? I just kinda need a consensus...

**Comment by Rinoka** [comment]
If I'm reading Kor correctly: Techwebs can be modified as you see fit, but anything that actively reduces the final time for full completion (currently it's 1h30m, discuss with Kor if you want it to be reduced to 1h) or otherwise negates it (ie frontloading all the cool shit at the start) will almost definitely be rejected.

**Comment by kevinz000** [comment]
what a meme 
i really need to finish my sensors that i've been trying to finish but couldn't for the last week.

**Comment by Buggy123** [comment]
If i can't remove the time to completion there's literally no point in ever touching it so...

**Comment by Rinoka** [comment]
That's a pretty bland take, @Buggy123...

As it is, total time to completion is one of the least relevant things in tgstation. There's a world to explore of making techwebs more involved to the station as a whole; there's also the possibility of exploring things so that there's a defined Research Choice people make, so the tone of the station is set to match (ie, if Phazon requirements were relaxed, but taking Phazon research took a significant amount of time, so it was a rarer but fun thing people picked).

_Total_ time to completion shouldn't shut down all of your possible ideas for it. If they do, you should rethink things.

(Plus, like Kor mentioned: The timegate can be reduced if you talk with them about it first.)

**Comment by KorPhaeron** [comment]
Rinoka gets it

Techwebs have poor UI and the nodes arent in the most interesting arrangement but it has a lot of potential if we fix it properly

**Comment by GuyonBroadway** [comment]
Say it with me now

"TIMEGATES ARE NOT BALANCE"

The main science nerf now comes from not being able to turn R&D into a gunfactory, seeing as that leaves them with no easy to get toys and STILL have to wait for mining.

Unlike say, botany who can use their skills to nullify their timegate R&D get nothing. They might as well afk for the first 20min in a round before doing thier job.

Who thought that having a mandatory and immutable wait period was a good idea? That is the dumbest and unfun shit I ever heard.

**Comment by duncathan** [comment]
I defy you to find a maintainer willing to merge this

**Comment by KorPhaeron** [comment]
@guyonbroadway

They are absolutely balance when we have rounds that usually last an hour or less and have antags that power up over time or have to wait X minutes to do their objective (war ops, clock cult)

I agree its not fun though which is why it isnt a job anymore and why people are working on ways to speed it up that arent as simple as printing stock parts

**Comment by GuyonBroadway** [comment]
Waiting is not gameplay. A delay does not balance gameplay, it removes it for a time. The end result is still as powerful but it turns up after the R&D department spends 20 min in soft jail.

**Comment by Bawhoppen** [comment]
Wow you guys are really upset that powergameR&D is gone

**Comment by KorPhaeron** [comment]
>a delay does not balance gameplay

Time is a resource especially in time limited rounds and definitely when multiple sides are racing eachothers timers what are you talking about. Roundstart gygaxes for example would crush ops/clock cult/blob/blood cult etc

But it takes a lot of time to build them so those antags have a chance to get their objectives done or get stronger themselves or compromise robo before the mechs exist.

This is not at all a concept unique to ss13 techwebs I am surprised you're arguing this

**Comment by GuyonBroadway** [comment]
>This is not unique to ss13 and I am surprised you are arguing it.

No kor that right there that does not address the issue that NOTHING THE PLAYER CAN DO can affect the mandatory job delay. 

I don't roll scientist to politly fucking wait for antags to have their fun before I get to do the job I signed up for.  

Hey why don't we make the chem dispensers start with 0 charge so antags can get thier objectives done before chem can manufacture healing chems and a spraybottle of acid, clf3 and neurotoxin.

Or why not nix the production and maturation times on botany so to get your super healing weed and killer tomato army has to sit tight till the nukies get the disk and go delta.

This time delay is shitty game design, any method of balance that involes not playing the damn game does not balance the game.

**Comment by Rinoka** [comment]
> that does not address the issue that NOTHING THE PLAYER CAN DO can affect the mandatory job delay.

which is why we're discussing ways to justify lowering the delay and discussing ways to make research not require _full completion_ to be effective; rather, research's choices being a way of round-themeing. both of those options still maintain the full-completion blockade but make things more interesting.

"i don't roll scientist to politely wait for antags to have their fun" yeah sorry go play sec. techwebs aren't about to go away and they probably won't be updated to a state you'll enjoy until they've been playtest-modified a solid amount.

as kor has been pointing out, though: _research is not a dedicated job anymore_. comparing it to any other job won't work because scientists are working with the side-toys right now, they are not expected to maintain research beyond touching buttons sometimes.

**Comment by KorPhaeron** [comment]
>that does not address the issue that NOTHING THE PLAYER CAN DO can affect the mandatory job delay.

>I agree its not fun though which is why it isnt a job anymore and why people are working on ways to speed it up that arent as simple as printing stock parts

>why people are working on ways to speed it up that arent as simple as printing stock parts

>why people are working on ways to speed it up


I literally posted that we are working on what you want, are you reading the posts you are arguing with?
